airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/actions.py

```diff
@@ -30,25 +30,4 @@ async def run_format(
         format_commands (List[str]): The list of commands to run to format the repository
     """
     format_container = container.with_exec(sh_dash_c(format_commands), skip_entrypoint=True)
-    return await format_container.directory("/src").export(".")
-
-
-def mount_repo_for_formatting(
-    dagger_client: dagger.Client,
-    container: dagger.Container,
-    include: List[str],
-) -> dagger.Container:
-    """Mounts the relevant parts of the repository: the code to format and the formatting config
-    Args:
-        container: (dagger.Container): The container to mount the repository in
-        include (List[str]): The list of files to include in the container
-    """
-    container = container.with_mounted_directory(
-        "/src",
-        dagger_client.host().directory(
-            ".",
-            include=include,
-        ),
-    ).with_workdir("/src")
-
-    return container
+    return await format_container.directory("/airbyte/repo").export(".")
```

Comment:
I'm confused about the renaming here - we're still mounting the code to `/src` [here](https://github.com/airbytehq/airbyte/pull/32999/files#diff-965ce6ff9499e8fecfc40eab13db5482e458a201ab7244588a3713062ebb7baeR59-R64). Happy with either name but don't think this will work as is.

Reply:
I think it's a typo. Running `fix`raises the following error on export:
```
QueryError: lstat /tmp/buildkit-mount200474152/airbyte/repo: no such file or directory
```

Reply:
Thanks for fixing this!